web/components/input/buttons/CheckboxGroup/Checkbox.sass

```diff
@@ -1,54 +1,49 @@
+.checkbox-group
+  @extend .flex, .wrap
+
 .checkbox
-  margin-left: 20px;
-  font-family: Apercu Pro;
-  font-size: 18px;
-  font-weight: bold;
-  line-height: 23px;
-  color: $primary;
-  position: relative;
-  margin-bottom: 8px;
-  cursor: pointer;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
+  @extend .user-select-none
+  margin-left: 20px
+  position: relative
+  margin-bottom: $spacing-s
 
   &:hover input:enabled ~ .checkmark
-    background-image: url('../assets/checkbox/hover.svg');
+    background-image: url('../assets/checkbox/hover.svg')
 
   input[type="checkbox"]
-    position: absolute;
-    opacity: 0;
-    cursor: pointer;
+    position: absolute
+    opacity: 0
+    cursor: pointer
 
     &:checked ~ .checkmark
-      background-image: url('../assets/checkbox/checked.svg');
+      background-image: url('../assets/checkbox/checked.svg')
       &:after
-        display: block;
+        display: block
 
     &:disabled
-      cursor: not-allowed;
-      opacity: 0.4;
+      cursor: not-allowed
       &~ .checkmark
-        cursor: not-allowed;
-        opacity: 0.4;
+        cursor: not-allowed
+        opacity: 0.4
         &~ .label-text
-          cursor: not-allowed;
-          opacity: 0.4;
+          cursor: not-allowed
+          opacity: 0.4
 
   .checkmark
-    position: absolute;
-    top: 0;
-    left: 0;
-    height: 24px;
-    width: 24px;
+    position: absolute
+    top: 0
+    left: 0
+    height: 24px
+    width: 24px
     background-image: url('../assets/checkbox/unchecked.svg')
+    cursor: pointer
 
     &:after
-      content: "";
-      position: absolute;
-      display: none;
+      content: ""
+      position: absolute
+      display: none
   
   .label-text
-    padding-left: 34px;
-    white-space: nowrap;
\ No newline at end of file
+    cursor: pointer
+    padding-left: 34px
```

Comment:
spacing variable

Reply:
Spacing class, actually - most of these properties can be declared using inline classes.

Reply:
Fixed